python/tvm/contrib/micro/meta_schedule/rpc_runner_micro.py

```diff
@@ -0,0 +1,243 @@
+                ),
+                timeout_sec=self.rpc_config.session_timeout_sec,
+            )
+            results.append(future)  # type: ignore
+        return results
+
+
+def _worker_func(
+    platform: str,
+    project_options: dict,
+    rpc_config: RPCConfig,
+    evaluator_config: EvaluatorConfig,
+    artifact_path: str,
+    device_type: str,
+    args_info: T_ARG_INFO_JSON_OBJ_LIST,
+) -> List[float]:
+
+    module_loader = micro.AutoTvmModuleLoader(
+        template_project_dir=micro.get_microtvm_template_projects(platform),
+        project_options=project_options,
+    )
+
+    remote_kw = {
+        "device_key": rpc_config.tracker_key,
+        "host": rpc_config.tracker_host,
+        "port": rpc_config.tracker_port,
+        "priority": 0,
+        "timeout": 100,
+    }
+    build_result = namedtuple("BuildResult", ["filename"])(artifact_path)
+
+    with module_loader(remote_kw, build_result) as (remote, mod):
+        dev = remote.device(device_type, 0)
+        f_prepare = ""
+        if evaluator_config.enable_cpu_cache_flush:
+            f_prepare = "cache_flush_cpu_non_first_arg"
+        time_f = mod.time_evaluator(
+            mod.entry_name,
+            dev,
+            number=evaluator_config.number,
+            repeat=evaluator_config.repeat,
+            min_repeat_ms=evaluator_config.min_repeat_ms,
+            f_preproc=f_prepare,
+        )
+
+        random_fill = remote.get_function("tvm.contrib.random.random_fill")
+        args = [nd.empty(x[2], x[1], dev) for x in args_info]
+        for arg in args:
+            random_fill(arg)
+        dev.sync()
+
+        costs = time_f(*args).results
+    return costs
+
+
+@contextmanager
+def get_rpc_runner_micro(
+    platform,
+    options,
+    session_timeout_sec: int = 300,
+    number: int = 3,
+    repeat: int = 1,
+    min_repeat_ms: int = 100,
+):
+    """Parameters
+    ----------
+    platform: str
+        The platform used for project generation.
+    project_options: dict
+        The options for the generated micro project.
+    session_timeout_sec: int
+        The session timeout. if the number of candidates sent to runner is larger
+        than the runner workers, increase the timeout.
+    number: int
+        The number of times to run the evaluator function for taking average.
+        We call these runs as one `repeat` of measurement.
+    repeat: int
+        The number of times to repeat the measurement.
+        In total, the function will be invoked (1 + number x repeat) times,
+        where the first one is warm up and will be discarded.
+        The returned result contains `repeat` costs,
+        each of which is an average of `number` costs.
+    min_repeat_ms: int
+        Minimum repeat time in ms. if the execution latency is too short,
+        increase the number of runs to the given time (in ms) to reduce the measurement error.
+    """
+    tracker_host = "127.0.0.1"
```

Comment:
Not sure if hardcoding the tracker information here is a good idea, what about use them as the default of another argument`rpc_config` for the function?